source/GameAction.cpp

```diff
@@ -214,6 +215,16 @@ void GameAction::LoadSingle(const DataNode &child, const string &missionName)
 			GameData::Missions().Get(toFail);
 		}
 	}
+	else if(key == "relocate")
+	{
+		for(const DataNode &grand : child)
+		{
+			if(grand.Token(0) == "planet" && grand.Size() > 1)
+				teleportPlanet = GameData::Planets().Get(grand.Token(1));
+			if(grand.Token(0) == "flagship only")
+				flagshipOnly = true;
```

Comment:
If we want to be able to use a location filter here instead of only having a single planet to teleport to in the future then we need to rethink this syntax now, because at current our location and other parameters about the relocation are at the same depth. That is, if we expand what is currently the `planet` keyword to be a location filter, the location filter would also take in the "flagship only" key, which we don't want. I'd suggest changing the syntax to the following:
```
relocate
	"flagship only" # Optional key.
	# Leaves room for other potential parameters.
	location # In the future a location filter, but at present only supports a planet to go to.
		planet "<planet>"
```

Reply:
New Syntax is:
```
relocate
	"flagship only" # optional
	location
		#location filter here
```